src/components/BulkEnrollmentPage/stepper/BulkEnrollmentSubmit.jsx

```diff
@@ -50,13 +57,15 @@ BulkEnrollmentAlertModal.propTypes = {
   isOpen: PropTypes.bool.isRequired,
   toggleClose: PropTypes.func.isRequired,
   enterpriseSlug: PropTypes.string.isRequired,
+  error: PropTypes.string.isRequired,
 };
 
 export const generateSuccessMessage = numEmails => `${numEmails} learners have been enrolled.`;
 
 const BulkEnrollmentSubmit = ({ enterpriseId, enterpriseSlug, returnToInitialStep }) => {
   const [loading, setLoading] = useState(false);
   const [checked, setChecked] = useState(true);
+  const [error, setError] = useState(true);
```

Comment:
should this be `useState('')` instead for initial state of 'error'? not sure if it matters, just noting that 'error' is a propTypes.string.required so worth a check ...

Reply:
Yep that makes sense. Or a null value. This is some copypasta error.